tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/BasicTupleTest.java

```diff
@@ -325,6 +327,13 @@ public void testTupleAsTupleFirstMember() {
         BRunUtil.invoke(result, "testTupleAsTupleFirstMember");
     }
 
+    @Test
+    public void testTupleName() {
+        Object returns = BRunUtil.invoke(result, "testTupleName", new Object[]{});
+        Type t = TypeUtils.getType(returns);
+        Assert.assertEquals(t.getName(), "T");
+    }
```

Comment:
Don't we need a runtime test for arrays also?

Reply:
Is there a way to test this since `ArrayValueImpl` doesn't have a type name?

Reply:
Please share the issue here too.

Reply:
#37470